components/NavigationDrawer/src/MDCBottomDrawerViewController.m

```diff
@@ -28,6 +28,7 @@ @interface MDCBottomDrawerViewController () <MDCBottomDrawerPresentationControll
 
 @implementation MDCBottomDrawerViewController {
   NSMutableDictionary<NSNumber *, NSNumber *> *_topCornersRadius;
+  BOOL _maskApplied;
```

Comment:
This flag doesn't seem to be gating application in `- bottomDrawerWillChangeState:` 
 Would you please rename it to reflect its behavior a bit more accurately? Something like `_maskAppliedInLayoutSubviews`?  I'm a little concerned a maintainer will not realize its purpose and will add it as a gating mechanism to other invocations of `- applyMask`.

Perhaps  `_isMaskAppliedFirstTime` would be accurate instead? It seems like it's meant for the case where the corner radius or drawer state isn't changed before presentation.